source/core/src/main/com/csse3200/game/components/tasks/bombship/BombshipCombatTask.java

```diff
@@ -4,28 +4,20 @@
 import com.badlogic.gdx.math.Vector2;
 import com.csse3200.game.ai.tasks.DefaultTask;
 import com.csse3200.game.ai.tasks.PriorityTask;
-import com.csse3200.game.entities.Entity;
-import com.csse3200.game.entities.factories.ProjectileFactory;
 import com.csse3200.game.physics.PhysicsEngine;
-import com.csse3200.game.physics.PhysicsLayer;
-import com.csse3200.game.physics.raycast.RaycastHit;
 import com.csse3200.game.services.GameTime;
 import com.csse3200.game.services.ServiceLocator;
 
-
-import java.util.ArrayList;
-
 /**
  * The AI Task for the Engineer entity. The Engineer will scan for targets within its detection range
  * and trigger events to change its state accordingly. This task must be called once the Engineer has
  * appropiately moved into position.
  */
 public class BombshipCombatTask extends DefaultTask implements PriorityTask {
-
     private static final int INTERVAL = 1; // The time interval for each target scan from the Engineer.
     private static final int PRIORITY = 3; // Default priority of the combat task when mobs are in range.
-    private static final short TARGET1 = PhysicsLayer.BOSS; // The type of targets that the Engineer will detect.
-    private static final short TARGET2 = PhysicsLayer.XENO;
+    // private static final short TARGET1 = PhysicsLayer.BOSS; // The type of targets that the Engineer will detect.
+    // private static final short TARGET2 = PhysicsLayer.XENO;
```

Comment:
Does the Bombship not target enemies, and is it handled somewhere else? why has this been commented out?

Reply:
I believe this combat task still needs some more functionalities; I just commented to check if it can fix the code smells.